tokens/dba.dk/icons.yml

```diff
@@ -5,9 +5,7 @@ icon:
   # Used for list-checked, hard coded stroke color value set to the equivalent of s-color-icon-selected (as it is not possible to have css vars as values in background images)
   list:
     checked: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' fill='none' viewBox='0 0 16 16'%3E%3Ctitle%3ECheckmark%3C/title%3E%3Cpath stroke='%230063fb' stroke-linecap='round' stroke-linejoin='round' stroke-width='1.5' d='M3.83 7.667 7.163 11l5.334-6'%3E%3C/path%3E%3C/svg%3E");
```

Comment:
Not sure if this is correct: `stroke='%230063fb'`

I think I forgot about this and just noticed this now

Reply:
Hmm... has the value of s-color-icon-selected changed?

Reply:
Ah, for DBA of course 👍

Reply:
Looks like it has changed for Finn as well... I'll check this...

Reply:
Nope, it was actually just DBA that was off... fixed now. 😄

Reply:
Good find! 😄 👍